x/programs/runtime/import_balance_test.go

```diff
@@ -14,6 +14,38 @@ import (
 	"github.com/ava-labs/hypersdk/x/programs/test"
 )
 
+func TestImportBalanceSendBalanceToAnotherProgram(t *testing.T) {
+	require := require.New(t)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	program := newTestProgram(ctx, "balance")
+	r := program.Runtime
+	stateManager := r.StateManager.(test.StateManager)
+	stateManager.Balances[program.Address] = 3
+
+	// create a new instance of the balance program
+	newInstanceAddress := codec.CreateAddress(0, ids.GenerateTestID())
+	programID := stateManager.AccountMap[program.Address]
+	r.StateManager.(test.StateManager).AccountMap[newInstanceAddress] = programID
```

Comment:
You could use public methods for more consistency

```suggestion
	programID, err := stateManager.GetAccountProgram(ctx, program.Address)
	require.NoError(err)
	r.StateManager.SetAccountProgram(ctx, newInstanceAddress, programID)
```